### DISCUSSION TITLE
How to use links in tables

### DISCUSSION THREAD

**Discussion by derrick56007** [h2oai/wave#1125] [Q&A] [answered]
Hello,

I am wondering how to get links in tables to work. I have the following code

`columns = [ui.table_column(name='edit_data', label='Edit', link=True)]`

and rows

`rows = [ui.table_row(name='test', cells=['https://www.google.com'])]`

however clicking on the "link" does not do anything. Is there something that I'm missing to get this to work?

Thanks and best,
Derrick

**Comment by mturoci** [comment]
Hi @derrick56007,

this is currently not supported, tracked in https://github.com/h2oai/wave/issues/428.

**Comment by lo5** [comment, marked as answer]
The workaround for now is to use plain tables inside markdown, either through `ui.markdown_card()` or `ui.text()`, if you're OK with not using `ui.table()`.

Example: https://wave.h2o.ai/docs/examples/text
